ci/playbooks/edpm_build_images/edpm_image_builder.yml

```diff
@@ -3,6 +3,19 @@
   ansible.builtin.import_playbook: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/playbooks/01-bootstrap.yml"
 - hosts: all
   tasks:
+    - name: Read hash from delorean.repo.md5 file
+      tags:
+        - edpm_build_img
+      slurp:
+        path: "{{ cifmw_basedir }}/artifacts/repositories/delorean.repo.md5"
```

Comment:
here https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/edpm_build_images.yaml#L19
```
cifmw_repo_setup_output: "/etc/yum.repos.d/"
```
is used. The delorean.repo.md5 will be generated in /etc/yum.repos.d/ directory.

Reply:
@raukadah In logs I can see delorean.repo.md5 in artifacts directory.

https://logserver.rdoproject.org/47/347/9b169ec193fe977729eb18eebae528e51d77b86d/github-check/periodic-edpm-build-push-images-centos-9-antelope/b355c95/ci-framework-data/artifacts/repositories/delorean.repo.md5

I was looking at code here https://github.com/openstack-k8s-operators/ci-framework/blob/0b3229de187e8184830ef3519ef1a070faab2053/ci_framework/roles/repo_setup/tasks/artifacts.yml#L21 and below logs of jobs, looks like `delorean.repo.md5` is created in artifacts directory

~~~
2023-07-07 06:54:05.559325 | controller | TASK [repo_setup : Dump full hash in delorean.repo.md5 file content={{ _get_hash.stdout | from_json | community.general.json_query('full_hash') }}
2023-07-07 06:54:05.559331 | controller | , dest={{ cifmw_repo_setup_basedir }}/artifacts/repositories/delorean.repo.md5] ***
2023-07-07 06:54:05.559337 | controller | Friday 07 July 2023  06:54:05 -0400 (0:00:00.444)       0:00:15.200 ***********
2023-07-07 06:54:06.303375 | controller | changed: [localhost]
~~~

Reply:
Thank you @Sandeepyadav93 for adding the logs , you are correct. I missed that part.